usr.sbin/inetd/inetd.h

```diff
@@ -0,0 +1,128 @@
+#ifndef _INETD_H
+#define _INETD_H
+
+#include <netinet/in.h>
+#include <sys/socket.h>
+#include <sys/time.h>
+#include <sys/un.h>
+
+#include <arpa/inet.h>
+
+#include <netdb.h>
+
+#include "pathnames.h"
+
+#ifdef IPSEC
+#include <netipsec/ipsec.h>
+#ifndef IPSEC_POLICY_IPSEC	/* no ipsec support on old ipsec */
+#undef IPSEC
+#endif
+#include "ipsec.h"
+#endif
+
+#define NORM_TYPE	0
+#define MUX_TYPE	1
+#define MUXPLUS_TYPE	2
+#define FAITH_TYPE	3
+#define ISMUX(sep)	(((sep)->se_type == MUX_TYPE) || \
+			 ((sep)->se_type == MUXPLUS_TYPE))
+#define ISMUXPLUS(sep)	((sep)->se_type == MUXPLUS_TYPE)
+
+#define	A_CNT(a)	(sizeof (a) / sizeof (a[0]))
+#define	TOOMANY		40		/* don't start more than TOOMANY */
+
+#define CONF_ERROR_FMT "%s line %zu: "
+
+/* Log warning/error with 0 or variadic args with line number and file name */
+#define IL0(prio, msg) syslog(prio, CONF_ERROR_FMT msg ".", \
+	CONFIG, line_number)
+
+#define ILV(prio, msg, args...) syslog(prio, CONF_ERROR_FMT msg ".", \
+	CONFIG, line_number, args)
+
+#define WRN0(msg) IL0(LOG_WARNING, msg)
```

Comment:
These can be done better with ```__VA_OPT__(,)```

Reply:
This is perfect, I'll remove ERR0 and WRN0 and use this with ERR and WRN instead.